src/js/mocks/server.js

```diff
@@ -0,0 +1,105 @@
+const jsonServer = require('json-server')
+const fs = require('fs');
+const server = jsonServer.create()
+const router = jsonServer.router('db.json')
+const middlewares = jsonServer.defaults()
+
+const VALID_USER = 'user';
+const VALID_PASSWORD = 'pass';
+const ACCESS_TOKEN = 'sdfjknwjnfwjnofonwewwf';
+const REFRESH_TOKEN = 'sdfjknwjnfwjnofonwewwf';
+
+
+server.use(middlewares)
+server.use(jsonServer.bodyParser)
+
+server.post('/api/accounts/login', (req, res) => {
+	const response = {detail: "Login or password invalid."}
```

Comment:
better to call this errorResponse so it's clear it's not the only response from this mock